app/jobs/regular/post_alert.rb

```diff
@@ -5,7 +5,7 @@ class PostAlert < Jobs::Base
 
     def execute(args)
       post = Post.find_by(id: args[:post_id])
-      if post&.topic
+      if post&.topic && !post.raw.blank?
```

Comment:
It's better to always use the positive `post.present?` is easier to understand than `!post.blank?`

```suggestion
      if post&.topic && post.raw.present?
```